Detectors/EMCAL/base/include/EMCALBase/Hit.h

```diff
@@ -42,24 +42,24 @@ namespace o2 {
       
       /// \brief Check whether the points are from the same parent and in the same detector volume
       /// \return True if points are the same (origin and detector), false otherwise
-      Bool_t operator==(const Point &rhs) const;
+      Bool_t operator==(const Hit &rhs) const;
       
       /// \brief Sorting points according to parent particle and detector volume
       /// \return True if this point is smaller, false otherwise
-      Bool_t operator<(const Point &rhs) const;
+      Bool_t operator<(const Hit &rhs) const;
       
       /// \brief Adds energy loss from the other point to this point
       /// \param rhs EMCAL point to add to this point
       /// \return This point with the summed energy loss
-      Point &operator+=(const Point &rhs);
+      Hit &operator+=(const Hit &rhs);
```

Comment:
Looking good ... I am just wondering here whether we need these kind of operators? Are you summing up hits? If yes how often is this done (it might be good to offer operators inline). We can address this later in a different PR.